first_lessons_dushanbe/array_sorting_binary_search.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+
+def list_sort(a):
+    if a is None:
+        return a
+    count = len(a)-1
+    while count >= 1:
+        ln = 0
+        for ln in range(count):
+            if a[ln] > a[ln+1]:
+                (a[ln], a[ln + 1]) = (a[ln + 1], a[ln])
```

Comment:
Could you please remove unnecessary `()`?

Reply:
done